v4/source/jobs.yaml

```diff
@@ -0,0 +1,67 @@
+  '/jobs/{job_id}/status':
+    get:
+      tags:
+        - jobs
+      summary: Get the status of a given job.
+      description: |
+          Gets the status of a single job,
```

Comment:
Oh actually can you add `__Minimum server version: 3.10__` on a new line? We need to start doing that for everything after 3.9

Also you have typo `,` instead of `.`